src/main/java/cart/auth/infrastructure/BasicAuthorizationExtractor.java

```diff
@@ -0,0 +1,27 @@
+package cart.auth.infrastructure;
+
+import cart.auth.dto.UserInfo;
+import cart.common.exceptions.AuthorizationException;
+import org.apache.tomcat.util.codec.binary.Base64;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Optional;
+
+public class BasicAuthorizationExtractor {
+
+    public static final String INVALID_USER_CREDENTIAL_ERROR = "유저 인증 정보가 잘못되었습니다.";
+    private static final String BASIC_TYPE = "Basic";
+    private static final String DELIMITER = ":";
+    String AUTHORIZATION = "Authorization";
```

Comment:
```suggestion
    private static String AUTHORIZATION = "Authorization";
```
직접 상수를 정의하지 않고 `HttpHeaders.AUTHORIZATION` 사용하는 방법도 있습니다!